lib/kino/smart_cell/chart_builder.ex

```diff
@@ -224,6 +227,7 @@ defmodule Kino.SmartCell.ChartBuilder do
   end
 
   defp build_arg_field(nil, _, _), do: nil
+  defp build_arg_field("count", _, _), do: [[aggregate: :count]]
```

Comment:
I can imagine `"count"` being a regular dataframe column, so for the value let's use something like `"__count__"`.

For the same reason it would make sense to make the label more distinctive. I actually like `COUNT(*)` the most because it matches SQL, but `COUNT` in all capitals is also alright.